crates/sui-core/src/consensus_adapter.rs

```diff
@@ -178,14 +189,34 @@ impl ConsensusAdapter {
         // certificate will be sequenced. So the best we can do is to set a timer and notify the
         // client to retry if we timeout without hearing back from consensus (this module does not
         // handle retries). The best timeout value depends on the consensus protocol.
-        match timeout(self.max_delay, waiter.wait_for_result()).await {
+        let back_off_delay =
+            self.max_delay + Duration::from_millis(self.delay_ms.load(Ordering::Relaxed));
```

Comment:
should we rename `max_delay` to `min_delay`?

Reply:
Or, half-max delay?